.envrc

```diff
@@ -218,23 +193,29 @@ fi
 make node-version-check
 
 if [ ! -x "node_modules/.bin/webpack" ]; then
-    info "You don't seem to have yarn packages installed."
-    advice_install_yarn_pkgs
+    warn "You don't seem to have yarn packages installed."
+    commands_to_run+=("make install-js-dev")
 fi
 
 PATH_add node_modules/.bin
 
-### Overrides ###
+# These are commands that can take a significant ammount of time
+if [ ${#commands_to_run[@]} -ne 0 ]; then
+    warn "Run the following commands to bring your environment up-to-date:"
+    for cmd in "${commands_to_run[@]}"; do
+        warn "    $cmd"
+    done
+fi
 
-cat <<EOF
-${green}${bold}direnv: SUCCESS!
-${reset}
-EOF
+### Overrides ###
 
-if [ -z "${SENTRY_SILENCE_DIRENV_NOTICE:-}" ]; then
+if [ "${log_level}" != "info" ]; then
```

Comment:
I'm dropping `SENTRY_SILENCE_DIRENV_NOTICE` and instead outputting the message if something needs attention.

I want to reduce the number of variables we support.